utils/run_test.sh

```diff
@@ -274,6 +274,7 @@ if [ -d "/mnt/daos" ]; then
     run_test src/common/tests/btree.sh dyn -s ${BTREE_SIZE}
 
     COMP="UTEST_csum"
+    run_test "${SL_PREFIX}/bin/cli_checksum_tests"
```

Comment:
Should this be SL_BUILD_DIR based?

Reply:
Oh, yes, probably. I was thinking the test_files_to_stash copied them into SL_PREVIX/bin, but I see that the other test files in run_test.sh are SL_BUILD_DIR based.